### DISCUSSION TITLE
Is there truly no way use both HTTP and HTTPS in Vercel?

### DISCUSSION THREAD

**Discussion by silasabbott** [vercel/vercel#7661] [Help]
I have a web app that uses localhost in an iframe to allow web developers to test their website against different device sizes. Since most dev servers don't have SSL, the only way to load localhost into an iframe in these cases is with plain old http. Is there really no way to do this in Vercel?

**Comment by paradise-point** [comment]
I got https SSL working on localhost with `vercel dev` using `mkcert` and `local-ssl-proxy` via the following executed by `npm run dev`

```
concurrently 'cp -v _dev.env .env && svelte-kit dev --host --port $PORT' 
'local-ssl-proxy --key localhost-key.pem --cert localhost.pem --source 3001 --target 3000'
```
